test/colony-funding.js

```diff
@@ -634,43 +689,68 @@ contract("Colony Funding", accounts => {
       const { colonyAddress } = logs[0].args;
       const newColony = await IColony.at(colonyAddress);
 
-      await checkErrorRevert(newColony.startNextRewardPayout(otherToken.address), "colony-reward-payout-invalid-total-tokens");
+      await newToken.setOwner(newColony.address);
+      await newColony.mintTokens(userTokens.toString());
+      await newColony.bootstrapColony([userAddress1], [userTokens.toString()]);
+      await newToken.transfer(newColony.address, userTokens.toString(), {
+        from: userAddress1
+      });
+
+      const result = await colony.getDomain(1);
+      const rootDomainSkill = result.skillId;
+      const colonyWideReputationKey = makeReputationKey(newColony.address, rootDomainSkill.toNumber());
+      const { key, value, branchMask, siblings } = await miningClient.getReputationProofObject(colonyWideReputationKey);
+      colonyWideReputationProof = [key, value, branchMask, siblings];
+
+      await checkErrorRevert(
+        newColony.startNextRewardPayout(otherToken.address, ...colonyWideReputationProof),
+        "colony-reward-payout-invalid-total-tokens"
```

Comment:
This require does not have both branches run, according to coverage, so this transaction is failing in an unintended place.